Project.toml

```diff
@@ -3,35 +3,36 @@ uuid = "2b0e0bc5-e4fd-59b4-8912-456d1b03d8d7"
 version = "3.0.1-DEV"
 
 [deps]
-JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
-REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 CSTParser = "00ebfdb7-1f24-5e51-bd34-a7502290713f"
 DocumentFormat = "ffa9a821-9c82-50df-894e-fbcef3ed31cd"
+JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
+JSONRPC = "b9b8584e-8fd3-41f9-ad0c-7255d428e418"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
 StaticLint = "b3cc710f-9c33-5bdb-a03d-a94903873e97"
-UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
-Tokenize = "0796e94c-ce3b-5d07-9a54-7f471281c624"
 SymbolServer = "cf896787-08d5-524d-9de7-132aaa0cb996"
+Tokenize = "0796e94c-ce3b-5d07-9a54-7f471281c624"
 URIParser = "30578b45-9adc-5946-b283-645ec420af67"
-JSONRPC = "b9b8584e-8fd3-41f9-ad0c-7255d428e418"
-
-[extras]
-Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
-Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
-LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
-SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
-JSON = "0.20, 0.21"
-julia = "1"
 CSTParser = "2.2"
 DocumentFormat = "2.2"
+JSON = "0.20, 0.21"
+JSONRPC = "1"
 StaticLint = "4.3"
-Tokenize = "0.5.7"
 SymbolServer = "4.2"
+Tokenize = "0.5.7"
 URIParser = "0.4.1"
-JSONRPC = "1"
+julia = "1"
+
+[extras]
+LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
+Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Could this be done with less reorganization? I think this PR only adds one package, right?

I also think (but not 100% positive) that PkgDev.jl (which I use for tagging versions) will change things back to whatever order it likes, so we shouldn't try to "fight" the order here, but just leave as is and ideally just add individual lines.

Reply:
sounds fair, `Pkg.add("Markdown")` automatically reorders things, so maybe we should have think about making PkgDev.jl to follow that though.